src/index.html

```diff
@@ -1,13 +1,292 @@
+                      >
+                    </a>
+                  </div>
+                </div>
+              </div>
+
+              <div class="products__description">Audio sunglasses</div>
+            </div>
+
+            <div class="grid">
+              <div class="grid__item--1-2 grid__item--tablet-2-5 grid__item--desktop-5-8">
+                <a href="#" class="button">View all</a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="about">
+        <div class="container">
+          <div class="grid">
+            <div class="grid__item--1-2 grid__item--tablet-1-3 grid__item--desktop-1-5">
+              <h2 class="about__title">
+                Why buy direct
+                <br>
+                from Bose
+              </h2>
+            </div>
+            <div class="grid__item--1-2 grid__item--tablet-4-6 grid__item--desktop-6-10">
+              <div class="about__content">
+                <div class="about__bullet-points">
+                  <p class="about__bullet-points--text">Free 2-day shipping and returns</p>
+                  <p class="about__bullet-points--text">90-day risk-free trial</p>
+                  <p class="about__bullet-points--text">World class customer service</p>
+                  <p class="about__bullet-points--text">My Bose account management</p>
+                </div>
+                <div class="about__description"
+                >A great product is more than what’s in the box. It’s a promise of premium performance, world-class support, and everything you expect from a trusted brand. It’s just one of many reasons why you’ll shop with confidence on Bose.com.
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="contacts">
+        <div class="container">
+          <h2 class="contacts__title">
+            Contact us
+          </h2>
+          <div class="grid">
+            <div class="grid__item--1-2 grid__item--tablet-1-3 grid__item--desktop-1-5">
+              <form action="#" method="get">
+                <label for="name"></label>
+                <input
+                  type="text"
+                  class="form-field"
+                  id="name"
+                  placeholder="Name"
+                >
+                <label for="email"></label>
+                <input
+                  type="email"
+                  class="form-field"
+                  id="email"
+                  placeholder="Email"
+                >
+                <label for="message"></label>
+                <textarea
+                  name="message"
+                  id="message"
+                  placeholder="Message"
+                  class="form-field form-field--message"
+                >
+                </textarea>
+              </form>
+              <button href="#" class="button button--primary">Send</button>
```

Comment:
Move the `button` element inside the `form` element and change its `type` attribute to `submit` to handle form submission properly.